172.21.0.1 - - [23/May/2022 00:54:21] "[35m[1mPOST /api/user HTTP/1.1[0m" 210 -
[2022-05-23 00:54:21,477] INFO in views: Login user: gozY5Hpa, ip: 172.21.0.1
23.05.2022 00:54:21.651 - app - ERROR - Can not create user: AX1wR6MszLr1hgDc, err: (pymysql.err.DataError) (1406, "Data too long for column 'surname' at row 1")
[SQL: INSERT INTO test_users (name, surname, middle_name, username, password, email, access, active, start_active_time) VALUES (%(name)s, %(surname)s, %(middle_name)s, %(username)s, %(password)s, %(email)s, %(access)s, %(active)s, %(start_active_time)s)]
[parameters: {'name': 'N2POUOtUOFaazqcoxfjHrU91vSPLXg0F5IhnpXpHmfGMt', 'surname': 'XMTnhcgUOwa2iTKFN4TqsyRYRvCpmw63nViCKKHoQ1LHPvjfNM1KL6f0DiWTyMggQEQf8Wkp8DfC8fxIqyG4ilw34Ov1NMU7mVfRhaaOWn8VzGlvZSio42KdJd8gw2X6gaTFCcluQETLtxm4jaDuM ... (2 characters truncated) ... gUlqbylWk50CxLGtlsypx4gThPu6Lvu0A7pcR1LhS9PgtscGV8tLindAm2mvjNAdNbyzJ3z1qWFPo8I7bHzgtArkgScH9PsAounxLv9INsmJZReHCBuJyeMci1eTPBN3AV1cIjNAFOxKdD8jIJRsd', 'middle_name': 'fcqd7wRlTSV0k6Uy4xZHxSp77ddItxRvESxJIb6y9I8858ScKGkGa1CEbVQ83gbElno8197Nc2DZ0gdz0Z5GYexKLQFISEzUoeWQxmflxRw9EeSxWBSmo2TaSFbYhnJzyNqV9B1qr5hQqJO3ikBoP ... (2 characters truncated) ... QP3767wBxiRZdFCpeAL8kYSqDLxCQeQXEFX9o6ILM9obvZqlNGtxohEPC77T9BN4lDUK7EUbPUWkDMEdBxX6t4vYqlZ1i2ybSseb7szuQNNIDx0thPFpNinPUZCOzukPOSngRQTHccOEenXd2AWA7', 'username': 'AX1wR6MszLr1hgDc', 'password': 'fdCicqwTFqwpNehGQqEaoaWaeZg28ekSftdBMC0l6nkifPIrn0WSkPmgAf17JdJoTog5cZdzj4DAAsILNXoG9aaUZ09ISLyXmAY4ZslMoqWPrDbF1E3B1vZWLefNaxLfvftCnI56ltr4Eq5ux7SsVFgKvBpag0IbHxxPnXvqiOfCkjMbL7CLCbcMw1xqwCY8QZWMYho9hRqHk9Pr1AzCWGfU5BeOeGcjc9efemksgMToG6pFSQy385fbTG4h1tR', 'email': '3ShL32cyYPryLAl6LYoUrnOsVBCWvo0XQ9KcDacOYmCJFwnkJM5gWMOs@mail.ru', 'access': '1', 'active': '0', 'start_active_time': None}]
(Background on this error at: https://sqlalche.me/e/14/9h9h)
[2022-05-23 00:54:21,651] ERROR in views: Can not create user: AX1wR6MszLr1hgDc, err: (pymysql.err.DataError) (1406, "Data too long for column 'surname' at row 1")
[SQL: INSERT INTO test_users (name, surname, middle_name, username, password, email, access, active, start_active_time) VALUES (%(name)s, %(surname)s, %(middle_name)s, %(username)s, %(password)s, %(email)s, %(access)s, %(active)s, %(start_active_time)s)]
[parameters: {'name': 'N2POUOtUOFaazqcoxfjHrU91vSPLXg0F5IhnpXpHmfGMt', 'surname': 'XMTnhcgUOwa2iTKFN4TqsyRYRvCpmw63nViCKKHoQ1LHPvjfNM1KL6f0DiWTyMggQEQf8Wkp8DfC8fxIqyG4ilw34Ov1NMU7mVfRhaaOWn8VzGlvZSio42KdJd8gw2X6gaTFCcluQETLtxm4jaDuM ... (2 characters truncated) ... gUlqbylWk50CxLGtlsypx4gThPu6Lvu0A7pcR1LhS9PgtscGV8tLindAm2mvjNAdNbyzJ3z1qWFPo8I7bHzgtArkgScH9PsAounxLv9INsmJZReHCBuJyeMci1eTPBN3AV1cIjNAFOxKdD8jIJRsd', 'middle_name': 'fcqd7wRlTSV0k6Uy4xZHxSp77ddItxRvESxJIb6y9I8858ScKGkGa1CEbVQ83gbElno8197Nc2DZ0gdz0Z5GYexKLQFISEzUoeWQxmflxRw9EeSxWBSmo2TaSFbYhnJzyNqV9B1qr5hQqJO3ikBoP ... (2 characters truncated) ... QP3767wBxiRZdFCpeAL8kYSqDLxCQeQXEFX9o6ILM9obvZqlNGtxohEPC77T9BN4lDUK7EUbPUWkDMEdBxX6t4vYqlZ1i2ybSseb7szuQNNIDx0thPFpNinPUZCOzukPOSngRQTHccOEenXd2AWA7', 'username': 'AX1wR6MszLr1hgDc', 'password': 'fdCicqwTFqwpNehGQqEaoaWaeZg28ekSftdBMC0l6nkifPIrn0WSkPmgAf17JdJoTog5cZdzj4DAAsILNXoG9aaUZ09ISLyXmAY4ZslMoqWPrDbF1E3B1vZWLefNaxLfvftCnI56ltr4Eq5ux7SsVFgKvBpag0IbHxxPnXvqiOfCkjMbL7CLCbcMw1xqwCY8QZWMYho9hRqHk9Pr1AzCWGfU5BeOeGcjc9efemksgMToG6pFSQy385fbTG4h1tR', 'email': '3ShL32cyYPryLAl6LYoUrnOsVBCWvo0XQ9KcDacOYmCJFwnkJM5gWMOs@mail.ru', 'access': '1', 'active': '0', 'start_active_time': None}]
(Background on this error at: https://sqlalche.me/e/14/9h9h)
172.21.0.1 - - [23/May/2022 00:54:21] "[35m[1mPOST /api/user HTTP/1.1[0m" 500 -
172.21.0.1 - - [23/May/2022 00:54:21] "[32mPOST /login HTTP/1.1[0m" 302 -
172.21.0.1 - - [23/May/2022 00:54:21] "GET /welcome/ HTTP/1.1" 200 -
